lib/debug-server.js

```diff
@@ -78,6 +94,11 @@ DebugServer.prototype._createSession = function(debugPort) {
   return Session.create(debugPort, this._config);
 };
 
+DebugServer.prototype._shouldAnnounceEndpoint = function() {
+  var addressInfo = this._httpServer.address();
+  return addressInfo.address === '0.0.0.0';
```

Comment:
I think we should support any non-localhost IPs too.

http://tools.ietf.org/html/rfc5735
```
127.0.0.0/8 - This block is assigned for use as the Internet host
loopback address.  A datagram sent by a higher-level protocol to an
address anywhere within this block loops back inside the host.  This
is ordinarily implemented using only 127.0.0.1/32 for loopback.  As
described in [RFC1122], Section 3.2.1.3, addresses within the entire
127.0.0.0/8 block do not legitimately appear on any network anywhere.
```

A proposed condition:

```js
return !/^127\.0\.0\./.test(addressInfo.address)
```

Please include a descriptive comment linking to the RFC.